packages/golden-layout/src/controls/Header.ts

```diff
@@ -773,31 +780,60 @@ export default class Header extends EventEmitter {
     this.tabDropdownContainer.hide();
     this.isDropdownShown = false;
 
+    // put it back in the header dom
```

Comment:
_Why_ are we putting it back in the header DOM? Why don't we just create the container when we show the list and then remove it completely when it's hidden?

Reply:
Which would be easier if this was react and not jquery 🤷